charts/vector/values.yaml

```diff
@@ -306,6 +306,36 @@ customConfig: {}
   #     encoding:
   #       codec: json
 
+# Default volumes. They can be manipulated.
```

Comment:
Lets use a comment like [promtail](https://github.com/grafana/helm-charts/blob/main/charts/promtail/values.yaml#L155C1-L157) has for these two new options.